lib/message_request.js

```diff
@@ -1,176 +1,176 @@
+    this.message = {}
   }
-};
+}
 
 Request.buildGet = function (scope, options) {
-  var message = { op: 'get', to: scope};
-  return new Request(message).setOptions(options);
-};
+  var message = { op: 'get', to: scope }
+  return new Request(message).setOptions(options)
+}
 
 Request.buildPublish = function (scope, value) {
-  var message = { op: 'publish', to: scope};
-  var request = new Request(message);
-  request.setAttr('value', value);
+  var message = { op: 'publish', to: scope }
+  var request = new Request(message)
+  request.setAttr('value', value)
 
-  return request;
-};
+  return request
+}
 
 Request.buildPush = function (scope, resource, action, value) {
-  var message = { op: 'push', to: scope};
-  var request = new Request(message);
-  request.setAttr('resource', resource);
-  request.setAttr('action', action);
-  request.setAttr('value', value);
+  var message = { op: 'push', to: scope }
+  var request = new Request(message)
+  request.setAttr('resource', resource)
+  request.setAttr('action', action)
+  request.setAttr('value', value)
 
-  return request;
-};
+  return request
+}
 
 Request.buildNameSync = function (scope, options) {
-  var message = { op: 'nameSync', to: scope};
-  return new Request(message).setOptions(options);
-};
+  var message = { op: 'nameSync', to: scope }
+  return new Request(message).setOptions(options)
+}
 
 Request.buildSet = function (scope, value, key, userType, clientData) {
-  var message = { op: 'set', to: scope};
-  var request = new Request(message);
-  request.setAttr('value', value);
-  request.setAttr('key', key);
-  request.setAttr('type', userType);
+  var message = { op: 'set', to: scope }
+  var request = new Request(message)
+  request.setAttr('value', value)
+  request.setAttr('key', key)
+  request.setAttr('type', userType)
   if (clientData) {
-    request.setAttr('clientData', clientData);
+    request.setAttr('clientData', clientData)
   }
 
-  return request;
-};
+  return request
+}
 
 Request.buildSync = function (scope, options) {
-  var message = { op: 'sync', to: scope};
-  var request = new Request(message).setOptions(options);
+  var message = { op: 'sync', to: scope }
+  var request = new Request(message).setOptions(options)
   if (request.isPresence()) {
-    request.forceV2Sync(options);
+    request.forceV2Sync(options)
   }
-  return request;
-};
+  return request
+}
 
 Request.buildSubscribe = function (scope, options) {
-  var message = { op: 'subscribe', to: scope};
-  return new Request(message).setOptions(options);
-};
+  var message = { op: 'subscribe', to: scope }
+  return new Request(message).setOptions(options)
+}
 
 Request.buildUnsubscribe = function (scope) {
-  var message = { op: 'unsubscribe', to: scope};
-  return new Request(message);
-};
+  var message = { op: 'unsubscribe', to: scope }
+  return new Request(message)
+}
 
 // Instance methods
 
 Request.prototype.forceV2Sync = function (options) {
```

Comment:
let's use the default argument here `options = {}` instead of mutating the input.